make sure abseil-cpp isnt picked up if present, fails the build

https://github.com/MapServer/MapServer/pull/6833

Index: flatgeobuf/include/flatbuffers/base.h
--- flatgeobuf/include/flatbuffers/base.h.orig
+++ flatgeobuf/include/flatbuffers/base.h
@@ -236,7 +236,7 @@ namespace flatbuffers {
       } }
       #define FLATBUFFERS_HAS_STRING_VIEW 1
     // Check for absl::string_view
-    #elif __has_include("absl/strings/string_view.h")
+    #elif __has_include("absl/strings/string_view.h") && (__cplusplus >= 201411)
       #include "absl/strings/string_view.h"
       namespace mapserver {
 namespace flatbuffers {
